### PR TITLE
Error reporting

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: src/main/resources/standardTestCases/missing_input_failure.test [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+files {
+  wdl: missing_input_failure/missing_input_failure.wdl
+  inputs: missing_input_failure/missing_input_failure.inputs
+}
+
+metadata {
+    workflowName: missing_input_failure
+    status: Failed
+    "failures.0.message": "Input evaluation for Call missing_input_failure.hello failed.: addressee:\n\tFile not found gs://nonexistingbucket/path/doesnt/exist"
+}
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/missing_input_failure.test]: Newline :)

File: src/main/resources/standardTestCases/missing_input_failure/missing_input_failure.inputs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "missing_input_failure.wf_hello_input": "gs://nonexistingbucket/path/doesnt/exist"
+}
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/missing_input_failure/missing_input_failure.inputs]: Newline

File: src/main/resources/standardTestCases/missing_input_failure/missing_input_failure.wdl [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+workflow missing_input_failure {
+  File wf_hello_input
+  
+  call hello {input: addressee = read_string(wf_hello_input) }
+  
+  output {
+    String salutation = hello.salutation
+  }
+}
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/missing_input_failure/missing_input_failure.wdl]: newline

**Review by cjllanwarne — COMMENTED** [review]
👍

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/121/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/121/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**